snprc_scheduler/src/client/components/TimelineDetails.jsx

```diff
@@ -128,32 +104,65 @@ class TimelineDetails extends React.Component {
+                                </div>
+                                <div className='col-sm-6 zero-side-padding'>
+                                    <div className='col-sm-5  zero-side-padding'><ControlLabel ref='timeline-draft-state'>Draft</ControlLabel></div>
+                                    <div className='col-sm-7 zero-side-padding'><FormControl type='checkbox' id='QcState' style={{width: '20px', height: '20px'}}
+                                                                           checked={timeline.QcState ? (timeline.QcState === 4) : false}
+                                                                           onChange={this.handleDraftCheck}
+                                                                           disabled={!timeline.RowId || timeline.IsInUse}
+                                    /></div>
+                                </div>
+
 
+                            </div>
                         </div>
-                        <div className='col-sm-4'>
+                        <div className='col-sm-3 zero-right-padding'>
                             <div className='row'>
-                                <div className='col-sm-4'><ControlLabel>Study Notes</ControlLabel></div>
+                                <div className='col-sm-4'><ControlLabel className='wrap-words input-wide'>Study Notes</ControlLabel></div>
                                 <div className='col-sm-7 zero-side-padding'><FormControl componentClass="textarea" id='Notes'
                                                 value={timeline.Notes ? timeline.Notes : ''}
                                                 onChange={this.handleChange}
-                                                disabled={!timeline.RowId}
+                                                disabled={!timeline.RowId || timeline.IsInUse}
                                 /></div>
                             </div>
                             <div className='row'>
-                                <div className='col-sm-4'><ControlLabel>Scheduler Notes</ControlLabel></div>
+                                <div className='col-sm-4'><ControlLabel className='wrap-words input-wide'>Scheduler Notes</ControlLabel></div>
                                 <div className='col-sm-7 zero-side-padding'><FormControl componentClass="textarea" id='SchedulerNotes'
                                                 value={timeline.SchedulerNotes ? timeline.SchedulerNotes : ''}
                                                 onChange={this.handleChange}
-                                                disabled={!timeline.RowId}
+                                                disabled={!timeline.RowId || timeline.IsInUse}
                                 /></div>
                             </div>
                         </div>
                     </div>
             )
-        // } else {
-        //     return <div>Please select a timeline to view it's details</div>
-        // }
```

Comment:
This commented out code need to be handled or its there for informational reasons?